src/scss/froala_blocks.scss

```diff
@@ -1,246 +1,23 @@
-@import './buttons';
```

Comment:
We should be using our own variables and mixins instead of the Boostrap default ones.

Reply:
🤔 I suggest a direction, where re-useing bootstrap button variables instead of custom btn overrides. The refactoring is clear and isolated here: 3709cc621eff29c7ab70808314f0b3c6b1e10051

Your project should follow variables introduces by the bootstrap origin.